detect-browser.js

```diff
@@ -19,17 +19,21 @@
     SmartPhone.getUserAgent = function() {
         return navigator.userAgent;
     };
-
+    
     SmartPhone.setUserAgent = function(userAgent) {
         this.userAgent = userAgent;
     };
 
+    SmartPhone._test = function(regex) {
```

Comment:
please create this function as a private function or skip this function from `SmartPhone.isAny` because in that function i am iterate all `SmartPhone` functions